assets/sass/styleguide.scss

```diff
@@ -1,88 +0,0 @@
-/* =========================================================================
```

Comment:
Inlining this into the layout as it's plain CSS and saves building another file in sass + keeps things in the same place.